tests/test_api.py

```diff
@@ -140,35 +140,37 @@ def test_folder_nested_files():
 
 
 def test_zipfile_unsupported_file():
-    with tempfile.TemporaryDirectory() as tmp_dir:
-        f = open(tmp_dir + "/unsupported_file.txt", "a")
+    with tempfile.TemporaryDirectory() as tmp:
+        f = open(tmp + "/unsupported_file.txt", "a")
         f.write("No geographical data")
         f.close()
-        with tempfile.NamedTemporaryFile(suffix=".zip") as tmp:
-            create_zip(tmp_dir, tmp)
-            result = geoextent.fromDirectory(tmp.name, bbox=True, tbox=True)
-            assert "bbox" not in result
-            assert "tbox" not in result
+        zip_path = tmp + "/zipfile.zip"
```

Comment:
This might break on some systems with different path formatting, please only use `os.path.join()`.

Reply:
Fixed !